source/features/repo-age.tsx

```diff
@@ -122,6 +137,9 @@ void features.add({
 	include: [
 		pageDetect.isRepoRoot
 	],
+	exclude: [
+		pageDetect.isEmptyRepoRoot
+	],
```

Comment:
@fregante I dont this will work since we have `waitForDomReady: false`

Reply:
It might, or it might not. I’m not particularly concerned about console errors on the rare empty repo. 

Anyway most of the changes to this file should be in their own PR, this is not a lint

Reply:
ok will send later tonight